src/utils/power_flow/power_flow.jl

```diff
@@ -172,6 +371,38 @@ function solve_powerflow!(system::System, nlsolve; args...)
     end
     x0 = zeros(N_BUS * 2)
 
+    #Create Jacobian structure
+    J0_I = Int64[]
+    J0_J = Int64[]
+    J0_V = Float64[]
+
+    for ix in a
+        F_ix_r = 2 * ix - 1
+        F_ix_i = 2 * ix
+
+        for j in neighbors[ix]
```

Comment:
We have the capability of taking into account BusTypes here to fill properly and avoid filling unnecessary "holes" of the Jacobian